pkg/network/protocols/postgres/protocol.go

```diff
@@ -196,7 +198,9 @@ func (*protocol) IsBuildModeSupported(buildmode.Type) bool {
 func (p *protocol) processPostgres(events []EbpfEvent) {
 	for i := range events {
 		tx := &events[i]
-		p.statskeeper.Process(NewEventWrapper(tx))
+		eventWrapper := NewEventWrapper(tx)
+		p.statskeeper.Process(eventWrapper)
+		p.telemetry.Count(&tx.Tx, eventWrapper)
```

Comment:
The signature of Count could have been simplified
note that eventWrapper is a wrapper around the transaction tx, so why do we need to provide a pointer to `tx.Tx` ass well?